### PR TITLE
Use random_bytes() instead of Utils::getRandomBytes().

### PR DESCRIPTION
**Pull request by minecrafter**
(This is similar to iTXTech/Genisys#1703, which I've authored.)

This PR changes all usages of `Utils::getRandomBytes()` to the builtin function `random_bytes()`. `Utils::getRandomBytes()` has been kept, but is simply wraps `random_bytes()`.
### Reason to modify
- No calls are made for secure random bytes, ever. Therefore, it mixes in only bad sources of entropy.
- Most of the "secure" random sources used read from the same ultimate source (/dev/urandom on Linux, macOS, and the BSDs), and the "weak" sources are extremely poor-quality.
- PHP 7 has a function to return cryptographically-secure random bytes, so the mess of code is not needed since we're using PHP 7.

### CODE REVIEW

**Comment by inxomnyaa** [comment]
_yawn_ what about that sha512 stuff

**Comment by minecrafter** [comment]
The old code was used to create a sequence of supposedly "random" bytes, but the data fed in is poor quality. This code is used for things like creating the default RCON password.

PHP 7 provides a way to get secure random bytes, this allows us to simplify the function.

File: src/pocketmine/Server.php [hunk 1]
```diff
@@ -1653,7 +1653,6 @@ public function __construct(\ClassLoader $autoloader, \ThreadedLogger $logger, $
 		}
 
 		$this->logger->info($this->getLanguage()->translateString("pocketmine.server.networkStart", [$this->getIp() === "" ? "*" : $this->getIp(), $this->getPort()]));
-		define("BOOTUP_RANDOM", @Utils::getRandomBytes(16));
```
Comment (inxomnyaa) [inline src/pocketmine/Server.php:1656]: what 'bout this?

Comment (minecrafter) [reply src/pocketmine/Server.php:1656]: ...turns out, this is used somewhere. I'll need to fix that.

Comment (inxomnyaa) [reply src/pocketmine/Server.php:1656]: I guess thats the Server id generated on start

Comment (minecrafter) [reply src/pocketmine/Server.php:1656]: It's not. It was used in `DebugMemoryCommand` but this was easily changed (and shouldn't affect the command at all).

**Comment by robske110** [comment]
Well, sorry but this is already included in my XBOX PR (yep,i know this has nothing to do with xbox, forgot to change  branches).

**Comment by robske110** [comment]
https://github.com/ClearSkyTeam/ClearSky/pull/676/commits/852bbda19dad86713f02de8fe49bcd6676f1b2d1

**Comment by minecrafter** [comment]
Sounds cool 👍

**Comment by robske110** [comment]
Should i change the author of that commit to you? Or just give some credit in the description of it. I'm sorry for the trouble. (I don't want to merge this, as i will die of the merge conflicts #git).

**Comment by minecrafter** [comment]
It doesn't matter.

**Comment by robske110** [comment]
Good, will modify the commit to give you credit, closing this now :)

**Comment by thebigsmileXD** [comment]
Actually this fails on some processors (ARM), for example android hosts

**Pull request closed without merging**